.idea/.gitignore

```diff
@@ -0,0 +1,6 @@
+# Default ignored files
```

Comment:
I suggest adding `.idea` to either your `.git_global` or local `.gitignore` files.